pyproject.toml

```diff
@@ -13,6 +13,12 @@ requests = "^2.31.0"
 [tool.poetry.group.jupyterhub.dependencies]
 jupyterhub = "^4.0.1"
 
+[tool.poetry.group.test]
+optional = true
+
+[tool.poetry.group.test.dependencies]
+ctadata = "^0.2.6"
+
 [tool.poetry.group.dev]
```

Comment:
hm what's the meaning of dev now if test is something else?

Reply:
Hum, you are right, maybe we should maybe rename it `standalone`. What do you think ?

Reply:
Sounds good.